docs/docusaurus.config.js

```diff
@@ -136,37 +147,49 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      colorMode: {
+        defaultMode: "light",
+        disableSwitch: true,
+        respectPrefersColorScheme: false,
+      },
       navbar: {
-        title: "Hydra: Head Protocol",
+        title: "Hydra Head Protocol",
         logo: {
-          alt: "Hydra: Head Protocol",
+          alt: "Hydra Head Protocol",
           src: "img/hydra.png",
+          style: { height: 27, marginTop: 2.5 },
           srcDark: "img/hydra-white.png",
         },
         items: [
           {
             to: "/docs",
-            label: "User Manual",
-            position: "left",
+            label: "User manual",
+            position: "right",
           },
           {
             to: "/docs/dev",
             label: "Developer Documentation",
-            position: "left",
+            position: "right",
           },
+          //commenting out for the time being
+          // {
+          //   to: "/core-concepts",
+          //   label: "Core Concepts",
+          //   position: "right",
+          // },
```

Comment:
Should: keep it until we find a better place for these

Reply:
@ch1bo Are you happy with us to merge it in if you're happy with it - it looks good on our end as well - or shall we await other approval before we do the merge to prod ?

Reply:
Let's merge the whole thing and we'll see it pushed to https://hydra.family/head-protocol/unstable. Then we can resolve some last things still (we would ping you if we need help) and publish it in the next days to normal https://hydra.family/head-protocol